.coveragerc

```diff
@@ -1527,6 +1527,12 @@ omit =
     homeassistant/components/zwave_me/sensor.py
     homeassistant/components/zwave_me/siren.py
     homeassistant/components/zwave_me/switch.py
+    homeassistant/components/myuplink/__init__.py
+    homeassistant/components/myuplink/api.py
+    homeassistant/components/myuplink/application_credentials.py
+    homeassistant/components/myuplink/coordinator.py
+    homeassistant/components/myuplink/sensor.py
+
```

Comment:
I think these should be in alphabetical order.

Reply:
Not sure i understand you point here, it seems alphabetical the list i added?

Reply:
myuplink should not come after zwave_me m < z

Reply:
It's added to the file by a tool build into the HA dev - so think's it fine :)